content/audio.md

```diff
@@ -129,6 +132,18 @@ This command will start PulseAudio after it's been stopped (this is not usually
 pulseaudio --start
 ```
 
+This command will check the status of PipeWire and show any errors if automatic restarts raised any errors:
+
+```
+systemctl --user status wireplumber
```

Comment:
This is checking WirePlumber, not PipeWire. There is a `pipewire.service` and a number of `pipewire-...` services that are separate from what you're referring to here. (It's fine if you've found WirePlumber to be the service that needs checked for errors, but the description for the command should reflect that.)